lib/services/auth_service.dart

```diff
@@ -87,8 +87,9 @@ class AuthenticationService {
       final User user = newUser.user;
       await addUserToFirebase(user.uid, user.displayName, user.email);
       if (newUser != null) {
-        Navigator.of(context)
-            .pushReplacementNamed(RouteConstant.DASHBOARD_SCREEN);
+        Navigator.of(context).pushReplacementNamed(
```

Comment:
As we are using StreamBuilder, Navigator won't work, try to do it main.dart using SharedPref